SecurityPkg/Tcg/Tcg2Pei/Tcg2Pei.c

```diff
@@ -1073,6 +1113,23 @@ PeimEntryMP (
   CreateTcg2PreUefiEventLogEntries ();
   // MU_CHANGE [END]
 
+  // MU_CHANGE [BEGIN] - Measure Firmware Debugger Enabled
+  CurrentDeviceState = GetDeviceState ();
+
+  if (CurrentDeviceState & DEVICE_STATE_SOURCE_DEBUG_ENABLED != 0) {
+    Status = MeasureFirmwareDebuggerEnabled ();
+    if (EFI_ERROR (Status)) {
+      DEBUG ((DEBUG_ERROR, "Failed to measure Firmware Debugger Enabled!\n"));
+      ASSERT_EFI_ERROR (Status);
+
+      PanicReport (__FILE__, __LINE__, "Failed to measure Firmware Debugger Enabled!\n");
+      // TODO cap PCR?

```

Comment:
@makubacki Please advise. Would it make sense to cap the PCR here? How is that done?

Reply:
My thought is that we would want to cap PCR 7 and not panic? One use case is the system is under debug legitimately and so you are okay with PCR 7 breaking, but still want to continue booting to debug whatever issue (which might be why the measurement failed :). But I would defer to @makubacki